test/src/hidden_ranges/hidden_line_ranges_test.dart

```diff
@@ -3,6 +3,11 @@ import 'package:flutter_code_editor/src/hidden_ranges/line_numbering_breakpoint.
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  final emptyTextBreakpointRanges = HiddenLineRanges(
+    breakpoints: [],
+    fullLineCount: 1,
+  );
```

Comment:
For some historical reason `Code.empty` has an empty list instead of a list of one empty line. I guess we also need a test that it produces `HiddenLineRanges` with `fullLineCount == 1` and not `0`.